core/object/script_language.h

```diff
@@ -136,6 +136,7 @@ class Script : public Resource {
 	virtual String get_source_code() const = 0;
 	virtual void set_source_code(const String &p_code) = 0;
 	virtual Error reload(bool p_keep_state = false) = 0;
+	virtual Error load_source_code(const String &p_path) = 0;
```

Comment:
Perhaps this method should also be bound in `ClassDB`?

https://github.com/godotengine/godot/blob/b8ed596769d2114797015833ce86f86ee872ecfa/core/object/script_language.cpp#L135-L136

Reply:
I'm not sure if it needs to be exposed to users.

Reply:
OK, you are right. If needed, we can always do it later.